lib/commands/context.js

```diff
@@ -192,6 +194,12 @@ async function setupExistingChromedriver (chromedriver) {
 }
 
 async function setupNewChromedriver (opts, curDeviceId, adbPort) {
+  // if a port wasn't given, pick a random available one
+  if (opts.chromeDriverPort === null || opts.chromeDriverPort === undefined) {
```

Comment:
This is because for the vast majority of cases, `==` causes problems where `===` does not.

We generally just use `if (opts.chromeDriverPort)`. There is also a method in `appium-support`

```
import { util } from 'appium-support';

// ...

if (util.hasValue(opts.chromeDriverPort) {
// ...
```